mod_papers/config.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+return array(
```

Comment:
This is good in the sense that we can use this to add a `start_year` or similar to make the module more general. At the moment we cut off the results at 2011 since tcqp started then, but others who use this module do not have that constraint.

With that being said, the `mod_papers.php` entry point is doing this already. Why not make a `$config` array there, add `update_time` and the `orcids` together and pass them both to `showPapers`?